packages/terra-site/src/Index.jsx

```diff
@@ -84,83 +86,86 @@ import ToggleButtonTestRoutes from 'terra-toggle-button/tests/nightwatch/ToggleB
-      <Route path="overlay" component={OverlayExamples} />
-      <Route path="popup" component={PopupExamples} />
-      <Route path="profile-image" component={ProfileImageExamples} />
-      <Route path="progress-bar" component={ProgressBarExamples} />
-      <Route path="responsive-element" component={ResponsiveElementExamples} />
-      <Route path="search-field" component={SearchFieldExamples} />
-      <Route path="slide-group" component={SlideGroupExamples} />
-      <Route path="slide-panel" component={SlidePanelExamples} />
-      <Route path="status" component={StatusExamples} />
-      <Route path="table" component={TableExamples} />
-      <Route path="text" component={TextExamples} />
-      <Route path="time-input" component={TimeInputExamples} />
-      <Route path="toggle" component={ToggleExamples} />
-      <Route path="toggle-button" component={ToggleButtonExamples} />
-    </Route>
-    <Route path="/tests" component={TestLinks} />
-    {AlertTestRoutes}
-    {ArrangeTestRoutes}
-    {BadgeTestRoutes}
-    {BaseTestRoutes}
-    {ButtonTestRoutes}
-    {ButtonGroupTestRoutes}
-    {DatePickerTestRoutes}
-    {DemographicsBannerTestRoutes}
-    {SlidePanelTestRoutes}
-    {I18nTestRoutes}
-    {IconTestRoutes}
-    {ImageTestRoutes}
-    {ResponsiveElementTestRoutes}
-    {ContentContainerTestRoutes}
-    {ListTestRoutes}
-    {ListItemTestRoutes}
-    {SingleSelectListTestRoutes}
-    {MultiSelectListTestRoutes}
-    {StatusTestRoutes}
-    {TableTestRoutes}
-    {TimeInputTestRoutes}
-    {MarkdownTestRoutes}
-    {ModalTestRoutes}
-    {ModalManagerTestRoutes}
-    {ProgressBarTestRoutes}
-    {PropsTableTestRoutes}
-    {GridTestRoutes}
-    {SearchFieldTestRoutes}
-    {PopupTestRoutes}
-    {FormTestRoutes}
-    {OverlayTestRoutes}
-    {LoadingOverlayTestRoutes}
-    {OverlayContainerTestRoutes}
-    {ToggleTestRoutes}
-    {ToggleButtonTestRoutes}
-    {HeadingTestRoutes}
-    {TextTestRoutes}
-    {SlideGroupTestRoutes}
-    {ProfileImageTestRoutes}
-  </Router>
+  <div className={styles.app} >
+    <Router history={hashHistory}>
+      <Redirect from="/" to="/site" />
+      <Route path="/site" component={App}>
+        <IndexRoute component={Home} />
+        <Route path="alert" component={AlertExamples} />
+        <Route path="app-delegate" component={AppDelegateExamples} />
+        <Route path="arrange" component={ArrangeExamples} />
+        <Route path="badge" component={BadgeExamples} />
+        <Route path="base" component={BaseExamples} />
+        <Route path="button" component={ButtonExamples} />
+        <Route path="button-group" component={ButtonGroupExamples} />
+        <Route path="content-container" component={ContentContainerExamples} />
+        <Route path="datepicker" component={DatePickerExamples} />
+        <Route path="demographics-banner" component={DemographicsBannerExamples} />
+        <Route path="form" component={FormExamples} />
```

Comment:
Adding onto Jake's comment below it may be easier to flatten form elements here. Rather than one form group that contains a chain of drilling into, it would be easier to find specific components if we provided first class links here to Input, Control, Field, etc...

I think there would be value into providing categories and grouping system into this sidebar. Several other UI toolkits group items. We're working on a grouping system for Kaiju and I think the hierarchy in Kaiju should closely match anything we come up with here, I'll post what we've started on. Any feedback would be welcome.

Atoms
- Heading
- Badge
- Image
- Progress Bar
- Text
- Button
- Icons
- Forms
  - Input
  - Control
  - Field
  - Fieldset
  - Select
  - etc

Molecules 
- Search Field
- Arrange
- Button Group
- Content Container
- Toggle
- Toggle Button
- Status

Organisms
- List
- Multi-Select List
- Single-Select List
- Demographics Banner
- Responsive Element
- Slide Panel
- Grid
- Table